yetanotherco/aligned_layer
# Fix: bump gnark version

# Bumps gnark versoin

## Description

Closes #1469 . We currently use `gnark v0.10.0` and `gnark-crypto v0.12.2` we should bump to `gnark v0.11.0` and `gnark-crypto v0.14.0`

## To Test
- Run a local devet and verify proofs land:
1. `make anvil_start_with_block_time`
2. `make aggregator_start`
3. `make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml` 
4. `make operator_start CONFIG_FILE=config-files/config-operator-1.yaml`
5. `make batcher_start_local`
6. `make telemetry_full_start`

Once you have the system running, send proofs

7. `make batcher_send_burst_groth16`
8. Go to jaeger `http://localhost:16686/search`
9. You should see the traces correctly created and task verified
<img width="939" alt="image" src="https://github.com/user-attachments/assets/e600fa00-5c11-4fdc-b028-5641890d86a8">

## Type of change

Please delete options that are not relevant.


## Checklist
